[CI] KibanaUserRoleIntegTests.testSearchAndMSearch failed

Two failures, one on a [PR build here](https://gradle-enterprise.elastic.co/s/erx56dqmrvdqa/tests/overview?outcome=FAILED) 
and one one a local run on "main" branch on commit 2d47cd723889e3c8b53cc46f4e208290178e1fad. Build scan is[ here](https://gradle-enterprise.elastic.co/s/62oztuju72voo/tests/overview?outcome=FAILED).

Reproduced locally with:

```
./gradlew ":x-pack:plugin:security:internalClusterTest" --tests "org.elasticsearch.integration.KibanaUserRoleIntegTests.testSearchAndMSearch" -Dtests.seed=DCA7742E96A49378
```

Error:
```
 2> java.lang.AssertionError: failure in bulk execution:
    [0]: index [logstash-20-12-2015], id [null], message [org.elasticsearch.ElasticsearchSecurityException: action [indices:admin/auto_create] is unauthorized for user [_system] with effective roles [_system], this action is granted by the index privileges [auto_configure,create_index,manage,all]]
    Expected: <false>
         but: was <true>
        at __randomizedtesting.SeedInfo.seed([DCA7742E96A49378:963EB296A550DAC6]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.elasticsearch.test.ESTestCase.assertThat(ESTestCase.java:2471)
        at org.elasticsearch.test.ESIntegTestCase.indexRandom(ESIntegTestCase.java:1827)
        at org.elasticsearch.test.ESIntegTestCase.indexRandom(ESIntegTestCase.java:1720)
        at org.elasticsearch.test.ESIntegTestCase.indexRandom(ESIntegTestCase.java:1704)
        at org.elasticsearch.test.ESIntegTestCase.indexRandom(ESIntegTestCase.java:1681)
        at org.elasticsearch.integration.KibanaUserRoleIntegTests.testSearchAndMSearch(KibanaUserRoleIntegTests.java:103)
```

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -236,15 +236,9 @@ tests:
 - class: org.elasticsearch.xpack.esql.expression.function.aggregate.AvgTests
   method: "testFold {TestCase=<double> #2}"
   issue: https://github.com/elastic/elasticsearch/issues/113225
-- class: org.elasticsearch.integration.KibanaUserRoleIntegTests
-  method: testGetMappings
-  issue: https://github.com/elastic/elasticsearch/issues/113260
 - class: org.elasticsearch.xpack.security.authz.SecurityScrollTests
   method: testSearchAndClearScroll
   issue: https://github.com/elastic/elasticsearch/issues/113285
-- class: org.elasticsearch.integration.KibanaUserRoleIntegTests
-  method: testGetIndex
-  issue: https://github.com/elastic/elasticsearch/issues/113311
 - class: org.elasticsearch.packaging.test.WindowsServiceTests
   method: test81JavaOptsInJvmOptions
   issue: https://github.com/elastic/elasticsearch/issues/113313
@@ -260,9 +254,6 @@ tests:
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=transform/transforms_force_delete/Test force deleting a running transform}
   issue: https://github.com/elastic/elasticsearch/issues/113327
-- class: org.elasticsearch.integration.KibanaUserRoleIntegTests
-  method: testValidateQuery
-  issue: https://github.com/elastic/elasticsearch/issues/113328
 - class: org.elasticsearch.xpack.security.support.SecurityIndexManagerIntegTests
   method: testOnIndexAvailableForSearchIndexAlreadyAvailable
   issue: https://github.com/elastic/elasticsearch/issues/113336
@@ -272,15 +263,9 @@ tests:
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=analytics/top_metrics/sort by scaled float field}
   issue: https://github.com/elastic/elasticsearch/issues/113340
-- class: org.elasticsearch.integration.KibanaUserRoleIntegTests
-  method: testFieldMappings
-  issue: https://github.com/elastic/elasticsearch/issues/113341
 - class: org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT
   method: test {yaml=reference/ccr/apis/follow/post-resume-follow/line_84}
   issue: https://github.com/elastic/elasticsearch/issues/113343
-- class: org.elasticsearch.integration.KibanaUserRoleIntegTests
-  method: testSearchAndMSearch
-  issue: https://github.com/elastic/elasticsearch/issues/113345
 - class: org.elasticsearch.action.bulk.IncrementalBulkIT
   method: testBulkLevelBulkFailureAfterFirstIncrementalRequest
   issue: https://github.com/elastic/elasticsearch/issues/113365

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -236,9 +236,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.expression.function.aggregate.AvgTests
   method: "testFold {TestCase=<double> #2}"
   issue: https://github.com/elastic/elasticsearch/issues/113225
-- class: org.elasticsearch.xpack.security.authz.SecurityScrollTests
-  method: testSearchAndClearScroll
-  issue: https://github.com/elastic/elasticsearch/issues/113285
 - class: org.elasticsearch.packaging.test.WindowsServiceTests
   method: test81JavaOptsInJvmOptions
   issue: https://github.com/elastic/elasticsearch/issues/113313
@@ -257,9 +254,6 @@ tests:
 - class: org.elasticsearch.xpack.security.support.SecurityIndexManagerIntegTests
   method: testOnIndexAvailableForSearchIndexAlreadyAvailable
   issue: https://github.com/elastic/elasticsearch/issues/113336
-- class: org.elasticsearch.xpack.security.authz.SecurityScrollTests
-  method: testScrollIsPerUser
-  issue: https://github.com/elastic/elasticsearch/issues/113338
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=analytics/top_metrics/sort by scaled float field}
   issue: https://github.com/elastic/elasticsearch/issues/113340

